VndbSharp/Models/VndbFlags.cs

```diff
@@ -63,7 +63,7 @@ public enum VndbFlags
 		///		<para>Valid on: Release, Character</para>
 		/// </summary>
 		[FlagIdentity("vn")]
-		VisualNovel = 1 << 7,
+		VisualNovels = 1 << 7,
```

Comment:
Missing the work around used with characters, in [VndbUtils](https://github.com/Nikey646/VndbSharp/blob/master/VndbSharp/VndbUtils.cs#L133)